dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -5,7 +5,7 @@
 
 
 # Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
-export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.05.11T0411'
```

Comment:
This will be eventually done dynamically be Jenkins job parameter configuration so the latest will be picked up automatically, right?